### ISSUE TITLE
bug: permission and user column is not shown properly

### ISSUE THREAD

**Issue by Jaehaks** [eza-community/eza#905]
1) eza 0.18.8
2) windows terminal + clink
3) windows 10

I started with eza to show directory list 
and I tested some commands, but permission  column and user column didn't be shown properly I tough, it is right?

![image](https://github.com/eza-community/eza/assets/26200835/94177f6e-2365-4bc3-98ce-acf6ff50829f)

**Comment by daviessm** [comment]
Users and permissions work differently on Windows, what are you expecting to see?

**Comment by Jaehaks** [comment]
> Users and permissions work differently on Windows, what are you expecting to see?

I understand it is different with linux,  Can I change this attribute to rwx format?

**Comment by daviessm** [comment]
No, eza does not support that format on Windows. It could possibly be changed to show the current user's permissions but it would be a lot of work.
